dask/array/routines.py

```diff
@@ -882,6 +885,268 @@ def histogram(a, bins=None, range=None, normed=False, weights=None, density=None
         return n, bins
 
 
+def _block_histogramdd(sample, bins, range=None, weights=None):
+    """Call numpy.histogramdd for a blocked/chunked calculation.
+
+    Slurps the result into an additional outer axis via [np.newaxis].
+    This new axis will be used to stack chunked calls of the numpy
+    function and add them together later.
+
+    Returns
+    -------
+    :py:object:`np.ndarray`
+        NumPy array with an additional outer dimension.
+
+    """
+    return np.histogramdd(sample, bins, range=range, weights=weights)[0][np.newaxis]
```

Comment:
Would it make sense to just slice here since we want that axis to be a singleton?

```suggestion
    return np.histogramdd(sample, bins, range=range, weights=weights)[0:1]
```

Reply:
I think it would make sense-- I took the slurp-into-new-axis idea from the implementation of `da.histogram`.

I did a quick IPython `%timeit` test with both methods and the results were in very good agreement. Would the slice be preferred?

Reply:
I've made the change to use a slice

Reply:
Thanks! 😄